translationCoreApps/VerseCheck
# VerseCheck Initial

### This PR Addresses:
Initial implementation of new design for Selection, Edit verse, Commenting, etc….

### To Test this PR:
Check out PR #46 on translationWords: https://github.com/translationCoreApps/translationWords_Check_plugin/pull/46


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/versecheck/1)
